packages/jsapi-shim/src/dh.types.ts

```diff
@@ -377,11 +386,9 @@ export interface Column {
   sort(): Sort;
 }
 export interface FilterValue {
-  ofString(input: object): FilterValue;
-  ofNumber(input: number): FilterValue;
-  ofNumber(input: LongWrapper): FilterValue;
-  ofNumber(input: object): FilterValue;
-  ofBoolean(input: boolean): FilterValue;
+  ofString(input: unknown): FilterValue;
+  ofNumber(input: unknown): FilterValue;
```

Comment:
`unknown`? We know it could be a `number` or `LongWrapper`, why not specify that?

Also, these are static methods in the namespace `dh.FilterValue`, they are not instance methods. Should re-jig this TypeScript to reflect that.

Reply:
In the source for these functions, they take an `Object` which is Java for basically any non-primitive, right?

`ofNumber` checks if it's a date, long, number, then tries parsing it

I moved these + the static filter condition methods to separate `FilterValueStatic` and `FilterConditionStatic` interfaces